ci_tools/bumpversion.sh

```diff
@@ -19,9 +19,7 @@ else
 
     # Bump the version in project and make commit back to master
 
-    bump2version patch --message "Bump version in Travis CI: {current_version} to {new_version} [ci skip]"
-	git config --local user.email "engineering@finbourne.com"
```

Comment:
I think we still need these (https://github.com/slemasne/repo-test/commits/master)

Reply:
I had to add them directly in the GHA YAML